src/test/java/DukeTest.java

```diff
@@ -0,0 +1,30 @@
+import duke.*;
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DukeTest {
```

Comment:
Consider following the test method naming convention `featureUnderTest_testScenario_expectedBehavior()`.